stdlib/ballerina-builtin/src/main/ballerina/ballerina/io/natives.bal

```diff
@@ -87,4 +87,4 @@ public native function <ByteChannel channel> close();
 @Description {value:"Function to check whether next record is available or not"}
 @Param {value:"channel: The TextRecordChannel to read text records from"}
 @Return {value:"True if the channel has more records; false otherwise"}
-public native function <TextRecordChannel channel> hasNextTextRecord () (boolean);
\ No newline at end of file
+public native function <DelimitedRecordChannel channel> hasNextTextRecord () (boolean);
```

Comment:
Shall we rename this function as well?

Reply:
@wggihan shall we open a new issue for the method rename. As this PR would basically fix the jira described. Otherwise it will be difficult to track